std_msgs/Header header
int32[] identities
geometry_msgs/Point32[] positions
geometry_msgs/Point32[] standardDeviations
int32[] widths
int32[] heights
geometry_msgs/Point32[] velocities
geometry_msgs/Point32[] averagedVelocities
